server/test/unit/server/webservices/views/test_search.py

```diff
@@ -78,6 +78,19 @@ class FakeSearchView(search.SearchView):
 
     @mock.patch('pulp.server.webservices.views.decorators._verify_auth',
                 new=assert_auth_READ())
+    @mock.patch('pulp.server.webservices.views.search.SearchView._parse_args')
+    def test_get_with_invalid_filters(self, mock_parse):
+        """
+        InvalidValue should be raised if param 'filters' is not json.
+        """
+        mock_parse.return_value = ({'mock': 'query'}, 'tuple')
+        search_view = search.SearchView()
+        mock_request = mock.MagicMock()
+        mock_request.GET = {'filters': 'invalid json'}
```

Comment:
I'd use a ``QueryDict`` here for authenticity. It might not make much difference, but we may as well keep tests as close to real circumstances as we can.